### ISSUE TITLE
This should also work on qemu that's shipped with RHEL

### ISSUE THREAD

**Issue by flyemsafe** [RedHat-EMEA-SSA-Team/hetzner-ocp4#45]
You can install coreos using the coreos iso by extracting the:

- efiboot.img 
- initramfs.img  
- vmlinuz

Creating a .treeinfo

```
cat /tmp/coreos/.treeinfo 
[general]
arch = x86_64
family = Fedora
platforms = x86_64
version = 29
[images-x86_64]
initrd = initramfs.img
kernel = vmlinuz
```
Then run virt-install with the **--location** argument.

```
#!/bin/bash

args='coreos.inst=yes '
args+='coreos.inst.install_dev=vda '
args+='coreos.inst.image_url=http://172.24.24.3:8080/pub/rhcos-42.80.20190828.2-metal-bios.raw.gz '
args+='coreos.inst.ignition_url=http://172.24.24.3:8080/pub/bootstrap.ign '
args+='ip=dhcp '
args+='rd.neednet=1'

virt-install --location /tmp/coreos --extra-args="${args}" --network network=ocp4 --name ocp4-compute-1 --memory 8192 --disk /var/lib/libvirt/images/ocp4-compute-1.qcow2

exit $?
```

* Reference: [coreos/coreos-installer](https://github.com/coreos/coreos-installer)

**Comment by rbo** [comment]
RHEL 8 works perfectly.

**Comment by rbo** [comment]
Added rhel7 support with commit: https://github.com/rbo/hetzner-ocp4/commit/9908ed086c2e1309223690a19adcf20e9e3fa746 

Will be merged with PR #75

For me issue is fixed.
